### PR TITLE
gnuplot: bumped for rebuild

### PR DESCRIPTION
**Pull request by kenmays**
Bumped for libvpx 1.11.0. Confirmed x86 build.
NOTE: Added gnuplot v6.0.0 recipe - requires libvpx 1.13.1 to install,, can build with libvpx 1.11.0.

### CODE REVIEW

**Comment by Begasus** [comment]
Better would be to check a build for "gd" and "gnuplot" (as gnuplot also needs gd). :)
And thus a revbump for both would be needed. That way we can clear the way to drop 1.11.0 in the end.

**Comment by kenmays** [comment]
GD fixed: https://github.com/haikuports/haikuports/pull/9956

File: sci-visualization/gnuplot/gnuplot-6.0.0.recipe [hunk 1]
```diff
@@ -73,7 +87,10 @@ BUILD()
 	aclocal
 	automake
 	runConfigure --omit-dirs binDir ./configure --bindir="$commandBinDir" \
+		--with-bitmap-terminals \
 		--without-cairo \
+		--with-lua=no \
+		--with-qt=qt6
```
Comment (Begasus) [inline sci-visualization/gnuplot/gnuplot-6.0.0.recipe:93]: qt6 here and qt5 in the requirements?

Comment (kenmays) [reply sci-visualization/gnuplot/gnuplot-6.0.0.recipe:93]: Allowed. Falls back to Qt5 if Qt6 not present during configure.

Comment (Begasus) [reply sci-visualization/gnuplot/gnuplot-6.0.0.recipe:93]: Doesn't explain why you use Qt6 here.

Comment (kenmays) [reply sci-visualization/gnuplot/gnuplot-6.0.0.recipe]: Fixed, in case the fallback gets removed in newer src revisions.

**Pull request merged**